e2e/playwright.config.ts

```diff
@@ -1,14 +1,14 @@
+import path from 'path';
```

Comment:
@munzah-shah
Can you explain why did you need to add this import?

Reply:
I was facing the following problem:
The page.goto() method was not able to access the baseURL from the playwright config. Tested it quite a few times, it only passed the tests when I was passing the complete URL to the method for example: "http://localhost:8000/learn" and would fail on only "/learn". 
On asking @Fahdkassim and exploring the config file in his PR, he suggested me to add this snippet into the config file to fix the problem.

Note: this problem did not occur in my last PR tho.